packages/new-polymath-issuer/src/components/DividendList/Container.tsx

```diff
@@ -38,6 +42,7 @@ export class DividendListContainerBase extends Component<Props> {
             <DividendListPresenter
               securityTokenSymbol={securityTokenSymbol}
               dividends={sortedDividends}
+              filterDividends={filterDividends}
```

Comment:
Here instead of adding and extra prop, you could just filter them in here and pass them already filtered here `dividends={sortedDividends}`. It's the container so it can do the filtering job! Presenter just receives the dividends already filtered and shows them.

Reply:
Thanks. I chose to use context Provider instead so in order to use hooks, I kept the logic in the Presenter

Reply:
So I don't think using a context is needed here. 
What I was suggesting is that you pass the `dividends` directly filtered. No extra prop `filterDividends`.
You can see that the `dividends` prop is already being sorted/filtered, the variable is called `sortedDividends` actually!
So instead of only being sorted, they could be sorted + filtered in the same place.
So no extra prop, context etc.